parser/grammar/presto-extensions.txt

```diff
@@ -111,7 +111,7 @@ void presto_aggregation_function():
 {
     "NUMERIC_HISTOGRAM"
   | "HISTOGRAM"
-  | "APPROEX_PERCENTILE"
+  | "APPROX_PERCENTILE"
```

Comment:
This fix should probably be a separate commit/PR?

Reply:
Done